import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/Flags.java

```diff
@@ -131,6 +131,11 @@
   )
   String destinationConfigInline = "";
 
+  @Parameter(
+      names = {"--store_compress_to_transfer_syntax"},
```

Comment:
We might add support for gzip in the future. So can we rename the flag to "store_compression_type" and document that this means a transfer syntax (for now)?

Reply:
Do you mean uploading zipped .dcm files? But they still could contain various transfer syntaxes, this seems to make more sense as a separate setting.

Reply:
Good point, let's leave as-is